src/components/CalendarMonthGrid.jsx

```diff
@@ -159,7 +163,29 @@ export default class CalendarMonthGrid extends React.Component {
         style={getTransformStyles(transformValue)}
         onTransitionEnd={onMonthTransitionEnd}
       >
-        {months}
+        {months.map((month, i) => {
+          const isVisible =
+            (i >= firstVisibleMonthIndex) && (i < firstVisibleMonthIndex + numberOfMonths);
+          return (
+            <CalendarMonth
+              key={`CalendarMonth-${i}`}
```

Comment:
Ideally this would not include the index at all and would be based off of unique datae. But what you have here is functionally equivalent to a plain index with unnecessarily added complexity of the string.